api/desecapi/tests/test_token_domain_policy.py

```diff
@@ -142,91 +151,117 @@ def test_policy_lifecycle(self):
         self.assertStatus(response, status.HTTP_200_OK)
-        self.assertEqual(response.data, self.default_data | data)
+        self.assertEqual(response.data, self.default_data | data | {"id": policy_id})
 
         ## can't create policy for other user's domain
-        data = dict(domain=self.other_domain.name, perm_dyndns=True, perm_write=True)
+        data = {
+            "domain": self.other_domain.name,
+            "subname": None,
+            "type": None,
+            "perm_write": True,
+        }
         response = self.client.create_policy(
             self.token, using=self.token_manage, data=data
         )
         self.assertStatus(response, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(response.data["domain"][0].code, "does_not_exist")
 
         ## another policy
-        data = dict(domain=self.my_domains[0].name, perm_dyndns=True)
+        data = {
+            "domain": self.my_domains[0].name,
+            "subname": None,
+            "type": None,
+            "perm_dyndns": True,
+            "perm_write": True,
+        }
         response = self.client.create_policy(
             self.token, using=self.token_manage, data=data
         )
         self.assertStatus(response, status.HTTP_201_CREATED)
+        policy_id = response.data["id"]
 
         ## can't create policy for the same domain
         with transaction.atomic():
             response = self.client.create_policy(
-                self.token,
-                using=self.token_manage,
-                data=dict(domain=self.my_domains[0].name, perm_dyndns=False),
+                self.token, using=self.token_manage, data=data
             )
         self.assertStatus(response, status.HTTP_409_CONFLICT)
 
         ## verify object creation
         response = self.client.get_policy(
-            self.token, using=self.token_manage, domain=self.my_domains[0].name
+            self.token, using=self.token_manage, policy_id=policy_id
         )
         self.assertStatus(response, status.HTTP_200_OK)
-        self.assertEqual(response.data, self.default_data | data)
+        self.assertEqual(response.data, self.default_data | data | {"id": policy_id})
 
         # List: now has two elements
         response = self.client.list_policies(self.token, using=self.token_manage)
         self.assertStatus(response, status.HTTP_200_OK)
         self.assertEqual(len(response.data), 2)
 
+        default_policy_id = None
+        for policy in response.data:
+            if (
+                policy["domain"] is None
+                and policy["subname"] is None
+                and policy["type"] is None
+            ):
+                default_policy_id = policy["id"]
+                break
+
         # Change
         ## all fields of a policy
-        data = dict(domain=self.my_domains[1].name, perm_dyndns=False, perm_write=True)
+        data = dict(
+            domain=self.my_domains[1].name,
+            subname=None,
+            type=None,
```

Comment:
also change subname/type (or change comment in l234)

Reply:
done